BootloaderCorePkg/Tools/CfgDataTool.py

```diff
@@ -917,7 +917,7 @@ def Main():
                             help='Configuration binary file')
     SignParser.add_argument('-o', dest='cfg_out_file', type=str, help='Signed configuration output binary file name to be generated', required=True)
     SignParser.add_argument('-k', dest='cfg_pri_key', type=str, help='Private key file (PEM format) used to sign configuration data', required=True)
-    SignParser.add_argument('-auth', dest='auth_type', type=str, help='Auth Type with RSA signing',  default = 'RSA2048SHA256')
+    SignParser.add_argument('-a', dest='hash_alg', type=str, help='Hash Type for signing',  default = 'SHA2_256')

```

Comment:
you should use "choice" keyword , pls refer to other examples

Reply:
will use choice keyword